packages/grafana-ui/src/components/BigValue/BigValueLayout.tsx

```diff
@@ -77,9 +77,16 @@ export abstract class BigValueLayout {
     return styles;
   }
 
-  getValueStyles(): CSSProperties {
+  getValueStyles(percentChange?: boolean): CSSProperties {
+    const valueFontSize = this.valueFontSize;
+    const fontSize = percentChange
+      ? this.props.parentOrientation === VizOrientation.Horizontal
```

Comment:
Can you describe why the VizRepeater orientation matters here? Struggle seeing when we would care, only the orientation (layout) of the BigValue itself should matter?